### PR TITLE
finished map method

### PR DESCRIPTION
**Pull request by Rahatk1**
Added the implementation of the map method. takes in number from user and doubles it

### CODE REVIEW

**Comment by Rahatk1** [comment]
I think theres a merge conflict bc of the file name, I will fix that

File: index.html
Comment (George-Sucuzhanay) [inline index.html:1]: Great work Rahat! I have some suggestions that would improve this. 

- its a bit unclear what the myMap does exactly, a description would be helpful
- explain why user would need to input nums into two separate input boxes, I was confused myself when testing it out
- i think highlighting the parameters used in the func would make it more clearer

**Review by George-Sucuzhanay — CHANGES_REQUESTED** [review]
I would highlight the following:
myEach is used for performing actions on each element of an array without creating a new array, while myMap is used for creating a new array by transforming each element based on a callback function

Also, @DavidA123777 @Kazi27 should we maintain html files w/ our names for our individual JS methods? Uniformity would be great!

File: index.html
Comment (Kazi27) [reply index.html:1]: I think ournames.html and ournames.js is fine no? Not entirely sure what we should do about the CSS though, should we make the website look more cohesive by having the same theme or each page different?

**Comment by Rahatk1** [comment]
Gotcha. Also for George basically what the map function does is it applies
a function to each element of the array For instance, my function is
multiplying each element by two. I’m just gonna directly add the updated
HTML, which explains what each function is doing to Rahat.

**Pull request closed without merging**